Feliz/Properties.fs

```diff
@@ -38,7 +39,67 @@ type AriaRelevant =
     /// Text is added to any DOM descendant nodes of the live region.
     | Text
 
+[<RequireQualifiedAccess; EditorBrowsable(EditorBrowsableState.Never)>]
+module PropHelpers =
```

Comment:
Please add `internal` modifier into this module?

Reply:
We can't use `internal` since the `prop` methods are inlined.